docs/source/maintainers.rst

```diff
@@ -0,0 +1,63 @@
+=====================
+Maintainer Guidelines
+=====================
+
+This page describes best practices for Dask maintainers.
+
+Thank you for helping make Dask a useful library and fostering a
+welcoming community.
+
+Merging pull requests
+=====================
+
+Pull requests should be reviewed
+--------------------------------
+
+Pull requests from non-maintainers should be reviewed and approved by at
+least one maintainer before being merged.
+
+Ideally, pull requests from maintainers should also be reviewed before
+merging. However, because reviewer bandiwdth is limited, mainainers will
+sometimes self-merge their own pull requests if the contents of the pull
+request is viewed as uncontroversial (e.g. typo fix). If a maintainer has
+a substantial pull request which hasn't received a review, and the maintainer
```

Comment:
```suggestion
a substantial pull request which hasn't received a review, they should ping at
least one team or individual who might be interested in the proposed change. 
If after some time (maybe 2 days) there is no response and the maintainer
```

Reply:
Ah, great. I made one minor tweak to the suggestion: adding the word "maintainer" to "or individual _maintainer_ who might be interested". Thoughts?

Reply:
In my mind the individual might not really be strictly a maintainer, I'm thinking an array expert for example, but I am happy with whatever.